src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -31,6 +31,11 @@ public class Deluge : TorrentClientBase<DelugeSettings>
             _proxy = proxy;
         }
 
+        public override void SetImportedCategory(DownloadClientItem downloadClientItem)
+        {
+            _proxy.SetLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
```

Comment:
Should this no-op if `TvCategory` and `TvImportedCategory` are the same? Also is the expectation that an empty category will clear the label in Deluge?

Reply:
I'll fix to check if the categories are the same before setting. I think clearing the label if blank is a reasonable expectation.